superset-frontend/src/features/datasets/AddDataset/EditDataset/UsageTab/index.tsx

```diff
@@ -124,11 +124,11 @@ const columns: ColumnsType<Chart> = [
 ];
 
 const emptyStateTableCSS = (theme: SupersetTheme) => css`
-  && th.ant-table-cell {
+  && th.antd5-table-cell {
```

Comment:
could try and use the standard tokens if/where possible, though we'll probably do a big pass to replace all `theme.colors\..*` soon